api/contentfulService.js

```diff
@@ -9,15 +9,15 @@ const client = createClient({
   host: CONTENTFUL_HOST,
 })
 
-export function fetchEntriesForContentType(contentType, config) {
+export function fetchEntriesForContentType(contentType, config, unpackItems = true) {
```

Comment:
I did that initially, but thought it would be nice to know that the config object only consisted of Contentful configs. But I think you're right - just a `false` in a function call is a bit random.